contracts/Vault.vy

```diff
@@ -1672,8 +1690,14 @@ def report(gain: uint256, loss: uint256, _debtPayment: uint256) -> uint256:
 
     # We have a loss to report, do it before the rest of the calculations
     if loss > 0:
+        pricePerShare: uint256 = self._shareValue(10 ** self.decimals)
```

Comment:
Could probably more succinctly write this as the following:
```suggestion
        if self.strategies[msg.sender].checkLoss:
            assert loss <= self.strategies[msg.sender].lossRatioLimit * strategy_totalDebt / MAX_BPS
        else:
            self.strategies[msg.sender].checkLoss = True
```

Reply:
It's either normalizing the loss to a ratio and then to a price, or calculating what the end effect would be and backing out from there.

Either way, it's saying "revert if >3% impact."